### ISSUE TITLE
Add response specification

### ISSUE THREAD

**Issue by mre** [slack-ruby/slack-api-ref#10]
The Slack API docs provide examples for each query response. These can range from [simple](https://api.slack.com/methods/pins.add) to [quite complex](https://api.slack.com/methods/im.history). For some use-cases - like code generation - it would be very helpful to include these response types in the slack-api-ref as well.
We could add the return type specification to each method or alternatively have a separate `types` folder which contains them. In any way, this addition would not break any current usages.
See [this repository](https://github.com/slack-rs/slack-api-schemas/tree/master/web) for reference.

**Comment by mre** [comment]
So, as an example, it would already be super helpful if we had the sample response from the documentation in the schema.

For [api.test](https://api.slack.com/methods/api.test) it could look like this:

```json
{
  "group": "api",
  "name": "api.test",
  "desc": "This method helps you test your calling code.",
  "args": {
    "error": {
      "required": false,
      "example": "my_error",
      "desc": "Error response to return."
    },
    "foo": {
      "required": false,
      "example": "bar",
      "desc": "example property to return."
    }
  },
  "response": {
    "sample": "{\"ok\":true,\"args\":{\"foo\":\"bar\"}}"
  },
  "errors": {
    "invalid_arg_name": "The method was passed an argument whose name falls outside the bounds of common decency. This includes very long names and names with non-alphanumeric characters other than _. If you get this error, it is typically an indication that you have made a very malformed API call.",
    "invalid_array_arg": "The method was passed a PHP-style array argument (e.g. with a name like foo[7]). These are never valid with the Slack API.",
    "invalid_charset": "The method was called via a POST request, but the charset specified in the Content-Type header was invalid. Valid charset names are: utf-8 iso-8859-1.",
    "invalid_form_data": "The method was called via a POST request with Content-Type application/x-www-form-urlencoded or multipart/form-data, but the form data was either missing or syntactically invalid.",
    "invalid_post_type": "The method was called via a POST request, but the specified Content-Type was invalid. Valid types are: application/json application/x-www-form-urlencoded multipart/form-data text/plain.",
    "missing_post_type": "The method was called via a POST request and included a data payload, but the request did not include a Content-Type header.",
    "request_timeout": "The method was called via a POST request, but the POST data was either missing or truncated."
  }
}
```

**Comment by dblock** [comment]
I like it, feel free to PR.

**Comment by mre** [comment]
Cool. Created a PR for that. #11

**Comment by dblock** [comment]
Closed via #11. Thanks!
